picard/webservice/__init__.py

```diff
@@ -149,6 +151,14 @@ def __init__(
 
         if not isinstance(url, QUrl):
             url = QUrl(url)
+
+        if queryargs is not None:
+            query = QtCore.QUrlQuery(url)
+            for k, v in queryargs.items():
+                # FIXME: check if encoding is correct
+                query.addQueryItem(k, str(v))
```

Comment:
I'm not sure encoding is correct here. Code is identical to `build_qurl()`.

See https://doc.qt.io/qt-6/qurlquery.html#QUrlQuery-1